docs/modules/ROOT/pages/vshn-managed/postgresql/major-upgrade.adoc

```diff
@@ -0,0 +1,53 @@
+= PostgreSQL Major Version Upgrade
+
+== Overview
+VSHNPostgreSQL supports major version upgrades up to the latest PostgreSQL 17.
```

Comment:
```suggestion
VSHNPostgreSQL supports major version upgrades up to the latest available version.
```

I'd leave the version number out here, this will be outdated yesterday.

Reply:
true